pypolyclip/pypolyclip.py

```diff
@@ -126,87 +134,90 @@ def single(x, y, nxy):
     x : int or float
         The x coordinates of the polygon corners
-    
+
     y : int or float
         The y coordinates of the polygon corners
 
     nxy : list, tuple, or `np.ndarray`
         The size of the pixel grid.
-    
+
     Returns
     -------
     xx : `np.ndarray`
         The x-pixel coordinates that have area (will be `int` dtype)
 
     yy : `np.ndarray`
         The y-pixel coordinates that have area (will be `int` dtype)
-    
+
     areas : `np.ndarray`
         The area projected onto a given pixel (will be `float` dtype)
 
     slice : list
         a slice object that will map between the input and output
         coordinates. This is *ONLY* returned for consistency between
-        the companion function `multi()`.  
+        the companion function `multi()`.
 
     Notes
     -----
     This is a Python driver to call JD Smith's polyclip.c code.
 
     """
-    
+
     # compute bounding box for the pixel
-    l = np.asarray(np.clip(np.floor(np.amin(x)), 0, nxy[0]),dtype=INT)
-    r = np.asarray(np.clip(np.floor(np.amax(x)), 0, nxy[0]),dtype=INT)
-    b = np.asarray(np.clip(np.floor(np.amin(y)), 0, nxy[1]),dtype=INT)
-    t = np.asarray(np.clip(np.floor(np.amax(y)), 0, nxy[1]),dtype=INT)
-    
+    l = np.asarray(np.clip(np.floor(np.amin(x)), 0, nxy[0]), dtype=INT)
+    r = np.asarray(np.clip(np.floor(np.amax(x)), 0, nxy[0]), dtype=INT)
+    b = np.asarray(np.clip(np.floor(np.amin(y)), 0, nxy[1]), dtype=INT)
+    t = np.asarray(np.clip(np.floor(np.amax(y)), 0, nxy[1]), dtype=INT)
+
     # get number of vertices for the polygon.  The C-code is expecting
     # this to be an array
-    nverts = np.full(1, len(x), dtype=INT) #np.array([len(x)], dtype=INT)
+    nverts = np.full(1, len(x), dtype=INT)  # np.array([len(x)], dtype=INT)
 
     # number of pixels that might be affected
-    npix = (r-l+1)*(t-b+1)
-    
+    npix = (r - l + 1) * (t - b + 1)
+
     # recast some things for C
     nclip = np.zeros(1, dtype=INT)
-    
+
     # output polygon indices
-    px_out = np.empty((nverts[0]+24)*npix, dtype=FLT)
-    py_out = np.empty((nverts[0]+24)*npix, dtype=FLT)
-    
+    px_out = np.empty((nverts[0] + 24) * npix, dtype=FLT)
+    py_out = np.empty((nverts[0] + 24) * npix, dtype=FLT)
+
     # main outputs (area, pixel coords and reverse indices)
     areas = np.empty(npix, dtype=FLT)
     inds = np.empty((npix, 2), dtype=INT)
-    ri_out = np.empty(npix+1, dtype=INT)
-    
+    ri_out = np.empty(npix + 1, dtype=INT)
+
     # call the pologyon clipper
-    polyclip.single(l, r, b, t,
-                    np.asarray(x,dtype=FLT),
-                    np.asarray(y,dtype=FLT),
-                    nverts, px_out, py_out, inds, nclip, areas, ri_out)
-   
+    polyclip.single(
+        l,
+        r,
+        b,
+        t,
+        np.asarray(x, dtype=FLT),
+        np.asarray(y, dtype=FLT),
+        nverts,
+        px_out,
+        py_out,
+        inds,
+        nclip,
+        areas,
+        ri_out,
+    )
```

Comment:
I really dislike `black` style formatting.  IMHO, the original was better, but this is a stylistic issue.